Entity/View.php

```diff
@@ -58,7 +58,7 @@ class View
     private $icon;
     
     /**
-     * @var string
```

Comment:
Nope it's a string, there is non hard link between the ems core data model and a user entity. Always a soft username string.

Reply:
This code is about the $options array. That is a json_array in doctrine. In php this is an array.
This is proven by it's usage on many places: 
$view->getOptions()['targetField']